src/app/dashboard/testDashboardLayout.ts

```diff
@@ -1,8 +1,9 @@
 import {Layout} from './layout';
 
 export const TEST_LAYOUT: Layout[] = [
-    {position : 1, name : 'Race', route : 'races'},
-    {position : 2, name : 'Spells', route : 'abilities'},
-    {position : 3, name : 'Classes', route : 'classes'},
-    {position : 4, name : 'Settings', route : 'settings'}
+    {position : 1, name : 'Home', route : 'home'},
+    {position : 2, name : 'Race', route : 'races'},
+    {position : 3, name : 'Spells', route : 'abilities'},
+    {position : 4, name : 'Classes', route : 'classes'},
+    {position : 5, name : 'Settings', route : 'settings'}
 ];
```

Comment:
This comment isn't for this file, I just added it here to say remove the src/app/home/home.component.css because there is nothing in it

Reply:
The file is necessary for code to run please make an exception as this design would have to be reflected on multiple components with empty css